### DISCUSSION TITLE
Binance Pay and Medusa Integration

### DISCUSSION THREAD

**Discussion by princessfrombabylon99** [tomk1v/binance-pay-module#3] [Ideas]
Hi toml1v, I realized that I could not get back to you with feedback. You might be wondering where I vanished to?  For some reasons, I decided to use [Medusa](https://github.com/medusajs). Will it be possible to integrate Binance Pay with it? Since you have good developer skills , Medusa is #1  on GitHub .

**Comment by tomk1v** [comment]
Hi princessfrombabylon99,

Thank you for reaching out! I appreciate your interest in a Binance Pay integration with Medusa. While I’m not directly available to implement this integration due to my current focus!

**Reply by princessfrombabylon99** [reply]
@tomk1v I am used to free open-source stuff . Hopefully someone will integrate Medusa and Binance Pay.  As Medusa has more stars on GitHub than Magento.
